codecov.yml

```
@@ -0,0 +1,31 @@
+codecov:
```

YAMLlint (1.37.1)

[error] wrong new line character: expected \n

(new-lines)